### PR TITLE
fix for terraform plugin build failure

### PR DESCRIPTION
**Pull request by tlake**
The plugin now does a proper comparison between the health check object received from `expandHealthCheck` and an empty initialized health check object. If they're equal (i.e. the health check field was completely unspecified in the `layer0.tf` file), the model is populated with our default values.

### CODE REVIEW

File: plugins/terraform/resource_load_balancer.go [hunk 1]
```diff
@@ -112,7 +112,8 @@ func resourceLayer0LoadBalancerCreate(d *schema.ResourceData, meta interface{})
 	ports := expandPorts(d.Get("port").(*schema.Set).List())
 	healthCheck := expandHealthCheck(d.Get("health_check"))
 
-	if !healthCheck {
+	blankCheck := models.HealthCheck{}
```
Comment (diemonster) [inline plugins/terraform/resource_load_balancer.go]: correct me if i'm wrong @zpatrick but I think the suggestion was something like
```
blankCheck := &models.HealthCheck{}
if healthCheck == *blankCheck {
```

Comment (tlake) [inline plugins/terraform/resource_load_balancer.go]: I don't think I'm catching the difference between this recommended
```
blankCheck := &models.HealthCheck{}
if healthCheck == *blankCheck {
```
and my own
```
blankCheck := models.HealthCheck{}
if healthCheck == blankCheck {
```

File: plugins/terraform/resource_load_balancer.go [hunk 2]
```diff
@@ -112,7 +112,7 @@ func resourceLayer0LoadBalancerCreate(d *schema.ResourceData, meta interface{})
 	ports := expandPorts(d.Get("port").(*schema.Set).List())
 	healthCheck := expandHealthCheck(d.Get("health_check"))
 
-	if !healthCheck {
+	if healthCheck == (models.HealthCheck{}) {
```
Comment (zpatrick) [inline plugins/terraform/resource_load_balancer.go]: This doesn't seem very canonical - could you update `expandHealthCheck` to return  a `*models.HealthCheck` and check `if healthcheck == nil` here?

Comment (tlake) [inline plugins/terraform/resource_load_balancer.go]: ✅

File: scripts/tlakemakefile [hunk 3]
```diff
@@ -0,0 +1,20 @@
+mocks:
```
Comment (zpatrick) [inline scripts/tlakemakefile]: forgot a .gitignore perhaps?

Comment (tlake) [inline scripts/tlakemakefile]: Ah shoot :p ✅

**Comment by zpatrick** [comment]
Going to move this into `develop` instead of `hotfix->master`. Reason being - we want these changes to be in `v0.9-rc2` and/or `v0.9` release

**Pull request merged**